cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/cluster-monitoring-view-balrog/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-monitoring-view
```

Comment:
name should match the parent folder name: `cluster-monitoring-view-balrog`

Reply:
done. thanks